JsonSchemas/CL_Schematic.json

```
@@ -117,6 +117,14 @@
                 "$ref": "./Components/CL_NameResolvedAsset.json"
             }
         },
+        "Schematics": {
+            "type": "array",
+            "description": "Automatic name resolved asset. Schematics to unlock when the schematic is purchased. Appended to the existing by default, use ClearSchematics to change this.",
```

```suggestion
            "description": "Automatic name resolved asset. Other schematics to unlock when the schematic is purchased. Note that these have *absolutely no visual representation within the game*, you will have to their confirm successful addition by testing or using ContentInspector. Appended to the existing by default, use ClearSchematics to change this.",
```

---

I think you're missing a word or something in that note.

---

`their` in the wrong place
```suggestion
            "description": "Automatic name resolved asset. Other schematics to unlock when the schematic is purchased. Note that these have *absolutely no visual representation within the game*, you will have to confirm their successful addition by testing or using ContentInspector. Appended to the existing by default, use ClearSchematics to change this.",
```